Simplenote/src/main/res/layout/nav_drawer_footer.xml

```diff
@@ -11,14 +11,15 @@
         android:layout_width="match_parent"
         android:layout_height="1px"
         android:layout_marginBottom="@dimen/padding_small"
-        android:background="?attr/dividerColor"/>
+        android:background="?attr/dividerColor" />
 
     <LinearLayout
         android:id="@+id/nav_settings"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:background="@drawable/selectable_background_simplestyle"
         android:clickable="true"
+        android:focusable="true"
```

Comment:
We definitely want to add the `android:focusable="true"` attribute to this view as you've done, but the rest of the changes to the file are just moving existing attributes around and are unnecessary.  We can revert those.

Reply:
Hey @theck13!
You're right, I wasn't sure about the formatting changes but I've reverted those!